upgrade/helpers/tasks.py

```diff
@@ -129,7 +132,13 @@ def sync_capsule_repos_to_upgrade(capsules):
         _add_additional_subscription_for_capsule(ak, capsuletools_url)
     # Publishing and promoting the CV with all newly added capsule, capsuletools, rhscl and
     # server repos combine
-    call_entity_method_with_timeout(cv.read().publish, timeout=2000)
+    logger.info("Content view publish operation has started successfully")
+    start_time = job_execution_time("CV_Publish")
+    call_entity_method_with_timeout(cv.read().publish, timeout=5000)
```

Comment:
Wht do you want  to update the timeout value from `2000` to `5000` ?

Reply:
To avoid further any timeout-related failure, we analyzed the log after sync.